src/apps/AppCenter/InstalledApps/AppContent.js

```diff
@@ -153,7 +160,9 @@ const AppContent = React.memo(({ repo, repoVersions, onRequestUpgrade }) => {
             <React.Fragment>
               <Heading2>Package Name</Heading2>
               <StyledLink
-                href={`https://etherscan.io/address/${repoName}#code`}
+                href={blockExplorerUrl('address', repoName, {
```

Comment:
So I just realized: on rinkeby we use our own instance of ENS so of course Etherscan thinks this is an invalid address :/.

We could use the `repo.repoAddress` instead, which is what the ENS name would resolve to.